ios/swift/JitsiSDKTest/src/ViewController.swift

```diff
@@ -41,9 +42,17 @@ class ViewController: UIViewController {
         JitsiMeet.sharedInstance().defaultConferenceOptions = defaultOptions
     }
     
+    override func viewWillTransition(to size: CGSize,
```

Comment:
Why is this necessary? Do all apps basically need to do this? I think a comment would help here.

Reply:
On rotation and size change we need to adjust the frame of the pip view and this is done using the pipViewCoordinator?.resetBounds(bounds: rect) method. All apps need to implement this if they support rotation.

Reply:
added comment